pycaret/containers/models/time_series.py

```diff
@@ -296,6 +296,50 @@ def _set_tune_grid(self) -> Dict[str, List[Any]]:
         return tune_grid
 
 
+class GrandMeansContainer(TimeSeriesContainer):
```

Comment:
You may have to override `eq_function` attr to make sure that it doesn't clash with the regular naive forecaster

Reply:
Filed a Bug. Will resolve in a separate PR